src/main/java/hudson/plugins/view/dashboard/test/TestTrendChart.java

```diff
@@ -149,18 +149,18 @@ public int compare(LocalDate d1, LocalDate d2) {
 
          if (run != null) { // execute only if job has builds
         	 LocalDate runDay = new LocalDate(
-        			 run.getTimeInMillis() - dateShift*60000, GregorianChronology.getInstanceUTC());
+        			 run.getTimeInMillis() - dateShift*60000l, GregorianChronology.getInstanceUTC());

```

Comment:
`dateShift`  and `60000` are both `int`s so multiplication could in theory overflow resulting in an incorrect calculation of the `run.getTimeInMillis() - dateShift*60000l.`  (`run.getTimeInMillis()` is a `long`).
By specifying the constant 60000 as a long this prevents the overflow.